ci-operator/step-registry/azure/deprovision/sp-and-custom-role/azure-deprovision-sp-and-custom-role-commands.sh

```diff
@@ -17,7 +17,7 @@ az --version
 
 # set the parameters we'll need as env vars
 AZURE_AUTH_LOCATION="${CLUSTER_PROFILE_DIR}/osServicePrincipal.json"
-if [[ -f "${CLUSTER_PROFILE_DIR}/installer-sp-minter.json" ]]; then
+if [[ "${AZURE_INSTALL_USE_MINIMAL_PERMISSIONS}" == "yes" ]] && [[ -f "${CLUSTER_PROFILE_DIR}/installer-sp-minter.json" ]]; then
```

Comment:
printing a debug info indicate that mini-perm is being used?

Reply:
sure, updated.